sonar-batch/src/main/java/org/sonar/batch/issue/DeprecatedIssueAdapterForFilter.java

```diff
@@ -82,7 +81,13 @@ public Integer line() {
   }
 
   @Override
+  @Deprecated
   public Double effortToFix() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)